tardis/analysis.py

```diff
@@ -262,7 +254,7 @@ def load_t_inner(self, iterations=None):
 
         for iter in iterations:
             t_inners.append(
-                hdf_store["model%03d/configuration" % iter].ix["t_inner"]
+                hdf_store[f"model{iter:003d}/configuration"].ix["t_inner"]
```

Comment:
Why has there been a change in the string formatting? What I mean is why is it `003d` instead of `03d`?

Reply:
> Why has there been a change in the string formatting? What I mean is why is it `003d` instead of `03d`?

This has got to be the silliest error done by me😅. Do let me know if there are any mistakes I must fix. Cheers !!

Reply:
Sure, Everything else seems to be Awesome. These things can be overlooked easily that's why I raised it here. Thank you for making the required changes 😄